### ISSUE TITLE
make a link to the Download page an icon on the front page

### ISSUE THREAD

**Issue by egonw** [bridgedb/BridgeDb#106]
@DeniseSl22, if you see my comment at https://github.com/bridgedb/BridgeDb/issues/103 you will see that apparently do not easily find the Download page. Fairly, the menu labels are a bit hard to read, as the contrast blue/white is perhaps too low. 

I want to propose a very prominent "Download" icon among the other main icons. To make the room, I would suggest the following:

- Remove the Plugins icons
- integration the info on the various plugins on the Download page
- integration the info on the various plugins on the Tutorial page

Do you agree?

**Comment by DeniseSl22** [comment]
Yes that could work! You mean download of the BridgeDb libraries, right? Not the local mapping files (I think these are displayed very clearly)? I have the "Framework" now as the first button people encounter, but that only explains the BridgeDb framework, doesn't link to the github page (that is done from the Developers page, but for clarity I can add the link on two places).

I don't think it's due to the colour scheme, but we can change that as well of course.

I'll start making some updates, and add a comment here when I'm done, okay?

**Comment by DeniseSl22** [comment]
Added:
- Main page: Framework and Libraries (iso Framework)
- Framework page: linkouts to github/bridgedb/bridgedb, and link to releases page on Github.

@egonw : could you check if this is what you had in mind?
